site/src/comparison.rs

```diff
@@ -357,28 +369,36 @@ impl ArtifactComparisonSummary {
     }
 
     /// The number of improvements that were found to be significant and relevant
-    pub fn number_of_improvements(&self) -> usize {
-        self.num_improvements
+    pub fn number_of_relevant_improvements(&self) -> usize {
+        self.num_relevant_improvements
     }
 
     /// The number of regressions that were found to be significant and relevant
-    pub fn number_of_regressions(&self) -> usize {
-        self.num_regressions
+    pub fn number_of_relevant_regressions(&self) -> usize {
+        self.num_relevant_regressions
     }
 
     /// Arithmetic mean of all improvements as a percent
```

Comment:
Now the comment should be probably changed.
```suggestion
    /// Arithmetic mean of relevant improvements as a percent
```